src/controllers/app/userController.js

```diff
@@ -53,21 +54,29 @@ exports.SignUp = function (req, res) {
 };
 
 exports.CheckCode = function (req, res) {
-  var { code, id } = req.body;
+  const { code, id } = req.body;
 
-  User.findById(id)
-    .populate("code")
-    .exec((err, foundedUser) => {
-      if (err) {
-        return res.status(500).json({ message: "server error" });
-      }
-      if (foundedUser === null) {
-        return res.status(404).json({ message: "user not found" });
-      }
-      if (foundedUser.code && foundedUser.code === code) {
-        return res.status(200).json({ message: "succecfully confirmed" });
+  User.findById({ _id: id }, (err, foundedUser) => {

```

Comment:
User.findById({ _id: id } 
change to
User.findById(id,

Reply:
done